lib/src/http.dart

```diff
@@ -210,8 +212,36 @@ class _PubHttpClient extends http.BaseClient {
 /// The [_PubHttpClient] wrapped by [httpClient].
 final _pubClient = new _PubHttpClient();
 
+/// A set of all hostnames for which we've printed a message indicating that
+/// we're waiting for them to come back up.
+final _retryMessages = new Set<String>();
```

Comment:
nit: consider `retriedHosts` or something along those lines - the name doesn't seem to match the description particularly well.

Reply:
Done.